cve_bin_tool/error_handler.py

```diff
@@ -192,6 +196,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     AttemptedToWriteOutsideCachedir: 28,
     SHAMismatch: 29,
     ExtractionFailed: 30,
+    ExtractionToolNotFound: 30,
```

Comment:
I'm ok with it as is but I wanted to make sure this was intentional that both ExtractionFailed and ExtractionToolNotFound will give the same error code.

Reply:
Yes it was intentional